staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -152,32 +152,74 @@ Get-CimInstance win32_pagefileusage | Format-List * | Out-File -Append "$ENV:TEM
+  $paths += "$ENV:TEMP\$timeStamp-disk-usage-all-drives.txt"
+}
+else {
+  Write-Host "Get-PSDrive command not available"
+}
+
 Write-Host "Collecting networking related logs"
 & 'c:\k\debug\collectlogs.ps1' | write-Host
 $netLogs = Get-ChildItem (Get-ChildItem -Path c:\k\debug -Directory | Sort-Object LastWriteTime -Descending | Select-Object -First 1).FullName | Select-Object -ExpandProperty FullName
 $paths += $netLogs
 $paths += "c:\AzureData\CustomDataSetupScript.log"
 
 # log containerd containers (this is done for docker via networking collectlogs.ps1)
-Write-Host "Collecting Containerd running containers"
+Write-Host "Collecting Containerd info from ctr"
+$ctrLogsDirectory = "$ENV:TEMP\$timeStamp-ctr-logs"
 $res = Get-Command ctr.exe -ErrorAction SilentlyContinue
 if ($res) {
-  Write-Host "Collecting Containerd running containers - containers"
-  & ctr.exe -n k8s.io c ls > "$ENV:TEMP\$timeStamp-containerd-containers.txt"
-  $paths += "$ENV:TEMP\$timeStamp-containerd-containers.txt"
+  New-Item -Type Directory $ctrLogsDirectory
+
+  Write-Host "Collecting ctr plugin ls"
+  & ctr.exe -n k8s.io plugin ls > "$ctrLogsDirectory\containerd-plugin.txt"
+
+  Write-Host "Collecting ctr containers"
+  & ctr.exe -n k8s.io c ls > "$ctrLogsDirectory\containerd-containers.txt"
+
+  Write-Host "Collecting ctr tasks"
+  & ctr.exe -n k8s.io t ls > "$ctrLogsDirectory\containerd-tasks.txt"
+
+  Write-Host "Collecting ctr content ls"
+  & ctr.exe -n k8s.io content ls > "$ctrLogsDirectory\containerd-content.txt"
 
-  Write-Host "Collecting Containerd running containers - tasks"
-  & ctr.exe -n k8s.io t ls > "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
-  $paths += "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
+  Write-Host "Collecting ctr image ls"
+  & ctr.exe -n k8s.io image ls > "$ctrLogsDirectory\containerd-image.txt"
 
-  Write-Host "Collecting Containerd running containers - snapshot"
-  & ctr.exe -n k8s.io snapshot ls > "$ENV:TEMP\$timeStamp-containerd-snapshot.txt"
-  $paths += "$ENV:TEMP\$timeStamp-containerd-snapshot.txt"
+  Write-Host "Collecting ctr snapshot ls"
+  & ctr.exe -n k8s.io snapshot ls > "$ctrLogsDirectory\containerd-snapshot.txt"
+
+  Write-Host "Collecting ctr snapshot tree"
+  & ctr.exe -n k8s.io snapshot tree > "$ctrLogsDirectory\containerd-snapshot-tree.txt"
+
+  Write-Host "Collecting ctr snapshot info for each snapshot"
+  $snapshotsList = & 'C:\Program Files\containerd\ctr.exe' -n k8s.io snapshot ls
+  foreach ($snapshot in $snapshotsList) {
+    $snapshotId = ($snapshot.Split(" ")[0])
+    $fileName = ($snapshotId.Split(":")[1])
+    if ($fileName.length -gt 0) {
+      & 'C:\Program Files\containerd\ctr.exe' -n k8s.io snapshot info $snapshotId > "$ctrLogsDirectory\containerd-snapshot-info-$fileName.txt"
+    }
+  }
+  $paths += $ctrLogsDirectory
 }
 else {
   Write-Host "ctr.exe command not available"
 }
 
+# collect disk usage information
+if (Test-Path "C:\tools\du.exe") {
```

Comment:
LGTM. The only thing that you need to update this path to `C:\aks-tools\DU\du.exe`

Reply:
I updated it directly.